x-pack/plugins/actions/README.md

```diff
@@ -272,6 +242,160 @@ const result = await actionsClient.execute({
+
+#### `subActionParams (getIncident)`
+
+| Property   | Description                           | Type   |
+| ---------- | ------------------------------------- | ------ |
+| externalId | The id of the incident in ServiceNow. | string |
+
+
+#### `subActionParams (getChoices)`
+
+| Property | Description                                                  | Type     |
+| -------- | ------------------------------------------------------------ | -------- |
+| fields   | An array of fields. Example: `[priority, category, impact]`. | string[] |
+
+---
+
+## Jira
+
+The [Jira user documentation `params`](https://www.elastic.co/guide/en/kibana/master/jira-action-type.html) lists configuration properties for the `pushToService` subaction. In addition, several other subaction types are available.
+### `params`
+
+| Property        | Description                                                                                                                                | Type   |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | ------ |
+| subAction       | The sub action to perform. It can be `pushToService`, `getIncident`, `issueTypes`, `fieldsByIssueType`, `issues`, `issue`, and `getFields` | string |
+| subActionParams | The parameters of the sub action                                                                                                           | object |
+
+#### `subActionParams (pushToService)`
+
+| Property | Description                                                                                                   | Type                  |
+| -------- | ------------------------------------------------------------------------------------------------------------- | --------------------- |
+| incident | The Jira incident.                                                                                            | object                |
+| comments | The comments of the case. A comment is of the form `{ commentId: string, version: string, comment: string }`. | object[] _(optional)_ |
+
+The following table describes the properties of the `incident` object.
+
+| Property    | Description                                                                                                      | Type                  |
+| ----------- | ---------------------------------------------------------------------------------------------------------------- | --------------------- |
+| summary     | The title of the issue                                                                                           | string                |
+| description | The description of the issue                                                                                     | string _(optional)_   |
+| externalId  | The id of the issue in Jira. If presented the incident will be update. Otherwise a new incident will be created. | string _(optional)_   |
```

Comment:
```suggestion
| externalId  | The ID of the issue in Jira. If present, the incident is updated. Otherwise, a new incident is created. | string _(optional)_   |
```